### DISCUSSION TITLE
[Vote ended on 2023-02-09] Ansible 8 Roadmap

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#194] [Votes done]
Please write +1 to accept the roadmap for Ansible 8 presented in https://github.com/ansible/ansible/pull/79598, i.e. merge that PR, and -1 to reject it.

Ref: #176

**Comment by felixfontein** [comment]
+1

**Comment by rooftopcellist** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by ssbarnea** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by acozine** [comment]
Assuming the folks who actually have to make the release happen have signed off on the schedule, I'm +1.
